lib/index.js

```diff
@@ -137,40 +134,42 @@ function configure() {
   });
 }
 
+async function listenHttps() {
+  const logger = bedrock.loggers.get('app');
```

Comment:
child logger? here and below?

Reply:
Fixed. I think this code was from before child loggers existed.